node/src/BaseClient.ts

```diff
@@ -479,7 +479,7 @@ export type BaseClientConfiguration = {
     defaultDecoder?: Decoder;
 };
 
```

Comment:
Removed decoder from ScriptOption and renamed ScriptOptions to ScriptOption for consistency.

Reply:
I think we need to keep old name `ScriptOptions` (plural), because there are multiple optional parameters inside - keys and args